test/e2e/e2e_test.go

```diff
@@ -333,6 +343,19 @@ func TestImportExport(t *testing.T) {
 
 	t.Logf("%v\n", inference)
 
+	var podEpochTime int64 = 1605312000
+	interpretations, err := runtime.getInterpretations("trader", podEpochTime, podEpochTime)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	assert.Equal(t, 1, len(interpretations))
+
+	err = snapshotter.Snapshot("interpretations.json", interpretations)
```

Comment:
This will be overwritten by the snapshot you do on line 411. You need to use the MultiSnapshot function for both.

Reply:
Specifically want the snapshot to be the same for both checks.